build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
This file has a lot of shellcheck errors around quoting strings. I believe it will fail if run from a path that includes a directory with a space in it.

Reply:
I fixed most of those. Only warnings are remaining, about no-used variables related to colors at the beginning.